### ISSUE TITLE
Dans occtax filtrer sur les relevés sans taxons

### ISSUE THREAD

**Issue by Amegilla** [PnX-SI/GeoNature#1318]
Maintenant que l'on peut créer des relevés sans taxons se serait utile de pouvoir les retrouver facilement avec un filtre spécial dans occtax. Vous en pensez quoi ?

**Comment by camillemonchicourt** [comment]
Ce n'est pas souhaitable de pouvoir saisir des relevés sans taxons dans Occtax (comme son nom l'indique). 
Mais avec la refonte du module, on n'a pas trouvé comment le bloquer complètement à cause de l'enregistrement continu.
Si jamais quelqu'un créé un relevé et ferme le formulaire alors on a un relevé sans taxons...

**Comment by Amegilla** [comment]
Oui je comprends bien, c'est pour cela que des outils pour faire la chasse aux relevés sans taxons serait utiles. Car si on ne peux pas les retrouver facilement, on ne peut pas s'en débarrasser :) ou les remplir.

**Comment by camillemonchicourt** [comment]
Il vaudrait mieux trouver une solution pour ne pas saisir de relevé sans taxon.

**Comment by Amegilla** [comment]
Ce n'est pas ce qui me dérange le plus, d'autant que ça peux servir pour saisir en différé. C'est surtout qu'ils soient oublié ensuite...

**Comment by DonovanMaillard** [comment]
Je reste dubitatif sur le fait de pouvoir les retrouver si du coup on peut pas identifier les relevés vides volontaires (saisies en attente etc) et les reliquats de saisies interrompues en vitesse. Si on veut saisir en différé il peut y avoir la solution de saisir aves sp. Avec un numéro de page de carnet à saisir ou ce genre de chose pais un relevé totalement vide sans aucune mention me semble moins pertinent. J'ai déjà retrouvé pas mal de relevés vides de mon côté, il faudrait presque une fonction avec un cron qui supprime les relevés vides créées il y a plus de x heures pour ne pas toucher aux saisies en cours etc... mais cest encore une fonction et une tache planifiée...

**Comment by jbrieuclp** [comment]
Quand tu poses des pièges sur plusieurs jours, tu peux d'office créer tes relevés vides tout de suite, au retour du terrain. Une fois tes pièges relevés et tes déterminations effectuées tu vas pouvoir compléter par les taxons les relevés initialement vides. C'est un usage qui est réel et apprécié dans ma structure.
Alors oui, je dois avoir des relevés sans taxons créés par erreurs, et alors ? Ils ne sont pas dans la synthèse, il ne sont pas dans les exports, ils ne sont pas dans la validation. Si vraiment c'est gênant pour l'utilisateur, il les supprimera, si vraiment c'est gênant pour l'administrateur qu'il les supprime.

**Comment by DonovanMaillard** [comment]
C'est pas ce à quoi occax est destiné à l'origine, et dans ce cas il faudrait plutôt viser un module dédié ? De mon coté, on a de la même manière des pièges sur lesquels on revient etc, je les ai traités dans le module de suivi pour avoir le type de piège, les dates de pose et de relevés etc dans les champs paramétrables.
C'est dommage que les usages particuilers ou les méthodologies des uns et des autres crée des données inachevées/incomplètes dans toutes les instances.

**Comment by Splendens** [comment]
C'est quand même pratique et confortable point de vue administration d'avoir un module que l'on peut utiliser pour plusieurs usages plutôt que de devoir gérer (voire développer et maintenir) un module pour chaque nouvel usage qui émergerait. A ce compte là on risque de se retrouver avec beaucoup de modules additionnels alors que les modules de base de GeoNature  peuvent faire le taf.

**Comment by camillemonchicourt** [comment]
A mon avis, il ne faut pas créer un module spécial pour cela, mais il ne faut pas non plus laisser Occtax en l'état avec des relevés sans taxons non souhaités. Il faut améliorer cette partie en bloquant réellement les relevés sans taxons. Ou alors réellement les permettre.
Mais là on les bloque à moitié et on en génère à l'insu de l'utilisateur sans le vouloir.

**Comment by DonovanMaillard** [comment]
Oui, apporter de la souplesse pour répondre à plusieurs besoins c'est forcément bien, avec des champs additionnels etc. Mais sans altérer le fonctionnement visé pour les besoins les plus classiques ou devoir supprimer les données incomplètes à la main. 

Encore une fois, aujourd'hui, le module de monitoring permet de faire ca sans développements particuliers pour des pièges qui sont posés et relevés à plusieurs reprise, type Barber/malaise etc.
